src/main/java/org/folio/s3/client/MinioS3Client.java

```diff
@@ -284,6 +286,21 @@ public List<String> list(String path) {
     }
   }
 
+  @Override
+  public Iterable<Result<Item>> iterableList(String path, int maxKeys, String startAfter) {
```

Comment:
The only idea. Does it make sense to return path of files as strings instead of sdk-related results? But it's up to you and your purposes.

Reply:
fixed